### PR TITLE
TimeFormat is changed.

### PR DESCRIPTION
**Pull request by ertugrul-aydin**

### CODE REVIEW

File: apps/samenvvv/components/HashtagCard/HashtagCard.tsx [hunk 1]
```diff
@@ -45,7 +45,10 @@ export const HashtagCard: FC<SliderHeroProps> = ({ item, type }) => {
             </Heading>
           )}
         </Box>
-        <Text>{item.date}</Text>
+        <Box>
+          <Icon as={FaCalendarDay} mr={2} />
+          <FormattedDate date={item.date as string} />
+        </Box>
```
Comment (7alip) [inline apps/samenvvv/components/HashtagCard/HashtagCard.tsx:51]: Nice catch! I forgot that we have `FormattedDate` component 😄 

FaCalendarDay looks a bit bold to me. Can you try `HiOutlineCalendar` instead.

Using HStack (horizontal Flex with gap) might be better for vertical alignment.
```ts
<HStack>
  <Icon />
  <FormattedDate>
</HStack>
```

**Review by 7alip — CHANGES_REQUESTED** [review]
Thank you!

File: apps/samenvvv/components/HashtagCard/HashtagCard.tsx (hunk 1 above)
Comment (ertugrul-aydin) [reply apps/samenvvv/components/HashtagCard/HashtagCard.tsx]: I updated icon and box elements 'HStack' instead.

**Review by 7alip — APPROVED** [review]
Thank you, it looks great now. Would you like to merge it?

**Pull request merged**